upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -468,6 +468,27 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		}
 	}
 
+	if b.cluster.Spec.Networking.Romana != nil {
```

Comment:
Should we test that k8s version is 1.6 or greater?

Reply:
I'm fine adding this. Is there an example to base it on?

Reply:
Probably best to add it to cluster validation and a similar example (where a networking mode is not supported on particular k8s versions) is here https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/validation/legacy.go#L400

(And I'm sure you could make romana work with k8s 1.5, but I agree that it probably isn't worth it :-) )

Reply:
@cgilmour did you make this change?

Reply:
Yes, [here](https://github.com/romana/kops/blob/558eb62e731be5f4af7f2c545cd860a9423ad918/pkg/apis/kops/validation/legacy.go#L396-L400)